lib/configure.js

```diff
@@ -479,19 +479,32 @@ PythonFinder.prototype = {
   },
 
   failNoPython: function failNoPython () {
-    var errmsg =
-        'Can\'t find Python executable "' + this.python +
-        '", you can set the PYTHON env variable.'
-    this.callback(new Error(errmsg))
+    const err = new Error(
+      '\n******************************************************************\n' +
+      `node-gyp can't use "${this.python}",\n` +
```

Comment:
It's quite convoluted code...
I'll give it another look in the morning but I can't promise I can find an elegant solution. This code comes after several rounds of geusses so it difficult to know what was found but deemed unsuitable or if nothing was found at all.